test/screens/new-ui.js

```diff
@@ -39,7 +39,7 @@ async function captureAllScreens() {
   const extPath = path.resolve('dist/chrome')
   driver = buildWebDriver(extPath)
   await driver.get('chrome://extensions-frame')
-  const elems = await driver.findElements(By.css('.extension-list-item-wrapper'))
+  const elems = await driver.findElements(By.css('* /deep/ extensions-item'))
```

Comment:
`/deep/` combinator is deprecated

Reply:
Darn. I will find an alternative, but coming up empty on solutions to getting the local extension id.

Reply:
@tmashuang its likely different on chrome and ff, eh?

Reply:
oh i see this is for chrome only atm

Reply:
there may be a better way to do this but
```
document.querySelector('extensions-manager').shadowRoot.querySelector('extensions-view-manager extensions-item-list').shadowRoot.querySelector('extensions-item')
```

Reply:
here is a single selector, annoying tho
```js
document.querySelector('extensions-manager::shadow extensions-view-manager extensions-item-list::shadow extensions-item')
```

Reply:
but `::shadow` is also deprecated, also `>>>`

Reply:
damn

Reply:
so then the multistep one above with `.shadowRoot` I guess